src/coreclr/jit/codegencommon.cpp

```diff
@@ -2377,11 +2377,7 @@ void CodeGen::genEmitMachineCode()
 
     compiler->unwindReserve();
 
-#if DISPLAY_SIZES
 
```

Comment:
the format job does not like these 3 empty lines, could you please delete 2?
You can fix it manually or apply the patch produced by the ci job (https://dev.azure.com/dnceng/_apis/resources/Containers/6976659/format.windows.x64.patch?itemPath=format.windows.x64.patch%2Fformat.patch) or run the format job locally ([doc](https://github.com/dotnet/jitutils/blob/main/doc/formatting.md)):
1. download https://github.com/dotnet/jitutils;
2. run bootstrap;
3. run `jit-format.bat --fix --coreclr path_to_runtime\runtime\src\coreclr`

Reply:
Thank you for your comment. Fixed it!